diff --git a/hloc/utils/database.py b/hloc/utils/database.py
index 870a8c4f..49976a58 100644
--- a/hloc/utils/database.py
+++ b/hloc/utils/database.py
@@ -173,7 +173,7 @@ def add_camera(self, model, width, height, params,
         return cursor.lastrowid
 
     def add_image(self, name, camera_id,
-                  prior_q=np.full(4, np.NaN), prior_t=np.full(3, np.NaN),
+                  prior_q=np.full(4, np.nan), prior_t=np.full(3, np.nan),
                   image_id=None):
         cursor = self.execute(
             "INSERT INTO images VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?)",
